### PR TITLE
Download metadata for all Crossref works

### PR DESCRIPTION
**Pull request by dhimmel**
As per the [figshare](https://doi.org/b48h):

> Queries began on 2017-03-21 and completed on 2017-04-02. The process died or stalled several times, but was restarted using the API's cursor. Accordingly, the works should represent the Crossref database as of 2017-03-21.

### CODE REVIEW

**Comment by dhimmel** [comment]
@stevemclaugh the Greene Lab has a pull request review policy before things go into the `master` branch. Feel free to review this PR, i.e. point out anything that doesn't look right or is confusing (or whatever else you think would be useful). No obligation of course... just that you will have a good handle on the content (xref https://github.com/greenelab/scihub/issues/4#issue-219459378)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  mongo:3.4.2
+```
+
+## Execution
+
+With mongo running, execute with the following commands:
+
+```sh
+# Download all works
+# To start fresh, use `--cursor=*`
```
Comment (kurtwheeler) [inline README.md:27]: You don't specify how to run using an existing cursor.

**Review by kurtwheeler — APPROVED** [review]
I only had one minor documentation comment which I was able to figure out the answer to by the time I finished my review. It might be nice to make that explicit, but isn't strictly necessary. Other than that this looks really good :)

File: README.md (hunk 1 above)
Comment (dhimmel) [reply README.md:27]: Added more doc in https://github.com/greenelab/crossref/pull/2/commits/d2213939e507ed007d2cffe6202bead839b1ecb6

**Pull request merged**